usr/bin/revpi-common

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+revpi_dev_type()
+{
+	return 0; # flat
```

Comment:
at moment only flat has BT, so just return 0.
later as cm4 connect equipped with  BT, change this function. 
for current connect/core/compact,  the service is skipped with condition failed, thus not be able to touch this function.